### PR TITLE
Create def script for easier lib def creation

### PR DESCRIPTION
**Pull request by Brianzchen**
Create a handy script inspired from #4042 that will create a basic structure for new type def configured to an appropriate lowest supported version as recommended in contribution guide.

Comes built with a lib def file plus a test file with base structure

Usage: `./create_def.sh lib-name 1.x.x`

### Skeleton File/Folder Structure
*sample when running `./create_def.sh test 1.x.x`*
Screen Shot 2021-08-28 at 4 31 14 pm

### Script Error Handling
Screen Shot 2021-08-23 at 11 55 58 pm

### CODE REVIEW

**Comment by Brianzchen** [comment]
@FezVrasta not exactly what you were asking for but what do you think? Should remove a lot of small mistakes with repetition when creating a new lib def

**Comment by GAntoine** [comment]
Could/should this be a command instead of a bash file? That way we can add it to the docs and help menu.

**Comment by Brianzchen** [comment]
> Could/should this be a command instead of a bash file? That way we can add it to the docs and help menu.

Sorry @GAntoine what do you mean? Do you want this to be a node command that lives in `cli/package.json` and then is exposed via a bash script as opposed to being a bash script itself? I was basically copying the way `./quick_run_def_tests.sh` gets used.

**Comment by GAntoine** [comment]
Yeah exactly! In the same way you'd run `flow-typed create-stub lib-name`, you could run `flow-typed create-def lib-name 1.x.x`. Then we could add it to the `help` menu and the wiki here on Github.

**Comment by Brianzchen** [comment]
@GAntoine Made a change, let me know if it's what you're looking for. Learnt more about the cli so that was good experience :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ version 4.1.0. **[You've followed all of the
 best practices for writing high quality definitions,](https://github.com/flow-typed/flow-typed/issues/13#issuecomment-214892914)**
 and now you'd like to contribute it:
 
+> Flow-typed comes with a handy script to simplify the file creation process `./create_def.sh` which takes two args, the library name and the version you'd like to type. In the :point_down: example that would be `./create_def.sh left-pad 4.x.x`.
```
Comment (AndrewSouthpaw) [inline CONTRIBUTING.md:52]: This'll need to be updated :)

**Review by AndrewSouthpaw — APPROVED** [review]
Looks great. @Brianzchen looks like maybe you didn't push the latest changes turning it into a CLI command?

File: CONTRIBUTING.md (hunk 1 above)
Comment (Brianzchen) [reply CONTRIBUTING.md:52]: Oh yep, let me fix this later today

Comment (Brianzchen) [reply CONTRIBUTING.md:52]: Actually I think it's fine as is. It just proxies to call `node cli/dist/cli.js create-def left-pad 4.x.x` so docs with how to contribute are still valid

**Pull request merged**